packages/workbox-strategies/CacheFirst.mjs

```diff
@@ -44,11 +44,13 @@ class CacheFirst {
    * @param {Object} options.fetchOptions Values passed along to the
    * [`init`](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch#Parameters)
    * of all fetch() requests made by this strategy.
+   * @param {Object} options.matchOptions [`CacheQueryOptions`](https://www.w3.org/TR/service-workers/#dictdef-cachequeryoptions)
```

Comment:
Could you link to `https://w3c.github.io/ServiceWorker/#dictdef-cachequeryoptions` here and in the other places? (The GitHub pages version of the spec docs tend to be more up to date.)